filament/backend/src/vulkan/VulkanBinder.cpp

```diff
@@ -352,10 +353,11 @@ void VulkanBinder::bindRasterState(const RasterState& rasterState) noexcept {
     }
 }
 
-void VulkanBinder::bindRenderPass(VkRenderPass renderPass) noexcept {
-    if (mPipelineKey.renderPass != renderPass) {
+void VulkanBinder::bindRenderPass(VkRenderPass renderPass, int subpass) noexcept {
```

Comment:
why is it not a bool (or enum)?

Reply:
I'm trying to be forward looking for our future, it's basically a zero-based index.  Let me rename it to `subpassIndex`.